Gemfile

```diff
@@ -12,6 +12,7 @@ gem 'rails', '4.1.8'
 
 gem 'activerecord-session_store'
 gem 'aes'
+gem "bugsnag"
```

Comment:
Are we ok to use double-quotes here in the Gemfile these days? If so ignore me.

Otherwise I'd like to whisper the following:

"quotes" (whispered).